Mathlib/MeasureTheory/Measure/MeasureSpace.lean

```diff
@@ -3559,6 +3559,21 @@ instance Add.sigmaFinite (μ ν : Measure α) [SigmaFinite μ] [SigmaFinite ν]
   refine' @sum.sigmaFinite _ _ _ _ _ (Bool.rec _ _) <;> simpa
 #align measure_theory.add.sigma_finite MeasureTheory.Add.sigmaFinite
 
+theorem SMul.sigmaFinite {μ : Measure α} [SigmaFinite μ] (c : ℝ≥0) :
+    MeasureTheory.SigmaFinite (c • μ) where
+  out' := ⟨{
+    set := spanningSets μ
+    set_mem := fun _ ↦ trivial
+    finite := by
+      intro i
+      simp only [smul_toOuterMeasure, OuterMeasure.coe_smul, Pi.smul_apply,
+        nnreal_smul_coe_apply]
+      refine Iff.mpr ENNReal.mul_lt_top_iff ?_
+      left
+      exact ⟨ENNReal.coe_lt_top, measure_spanningSets_lt_top μ i⟩
+    spanning := iUnion_spanningSets μ
+  } ⟩
```

Comment:
I *think* the indentation convention is the following:
```suggestion
  out' := 
  ⟨{ set := spanningSets μ
      set_mem := fun _ ↦ trivial
      finite := by
        intro i
        simp only [smul_toOuterMeasure, OuterMeasure.coe_smul, Pi.smul_apply,
          nnreal_smul_coe_apply]
        refine Iff.mpr ENNReal.mul_lt_top_iff ?_
        left
        exact ⟨ENNReal.coe_lt_top, measure_spanningSets_lt_top μ i⟩
      spanning := iUnion_spanningSets μ }⟩
```

Reply:
Ah, but I wrote it wrong :facepalm: All fields should of course have the same indentation as the first one. Sorry for that.

Reply:
I'll blame it on GitHub not having a monospace font when writing suggestions...

Reply:
Ah, that's why it broke. I'll try to edit it in VS Code so I can see the spacing...